src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -102,4 +102,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       Include="$(MSBuildThisFileDirectory)..\codestyle\vb\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.dll"
       IsImplicitlyDefined="true" />
   </ItemGroup>
+  
+  <!-- Build Properties Exposed to CodeStyle Analyzers -->
+  <ItemGroup Condition="$(EnforceCodeStyleInBuild)>
```

Comment:
Are you missing quotes on the condition? Also, I am wondering if we should make this non-conditional so it enables this feature in IDE even when users are not enforcing code style on build. @jmarolf your thoughts?

Reply:
I have fixed the quotation. Would the addition in VS [here](https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/271377) not enable the IDE scenarios?

Reply:
Hmm even with that PR, the condition disables it explicitly if user directly installs the CodeStyle NuGet package: https://github.com/dotnet/roslyn/blob/master/src/CodeStyle/CSharp/CodeFixes/build/Microsoft.CodeAnalysis.CSharp.CodeStyle.props

I think we should keep it simple:
1. Remove the condition here, so this feature works in VS + command line for SDK style projects for all the 3 analyzer sources:
   1. Default VS install with built-in code style analyzers coming from VS
   2. User sets `EnforceCodeStyleInBuild` to true and consumes code style analyzers from the .NET SDK
   3. User installs CodeStyle nuget package to consume code style analyzers from a specific version of the package.
2. Don't add support for this feature in non-SDK style projects, users would be asked to migrate to SDK style projects to get this feature.

Reply:
With the above, you should not need the VS side PR. Also note that Microsoft.CodeAnalysis.Targets is not imported during `dotnet build`, so it will cause inconsistency where it works somewhere, but not everywhere.

Reply:
Ah I see. I wasn't aware this was explicitly disabled on installing the package. Removing the condition here makes sense.
On the note of Microsoft.CodeAnalysis.Targets, do we not intend to support Non-SDK style projects in VS since we wouldn't be able to support them in CLI build scenarios?

Reply:
> do we not intend to support Non-SDK style projects in VS 

That is an open question that probably @mikadumont should answer.